exercises/alphametics/canonical-data.json

```diff
@@ -96,6 +96,23 @@
             "S": 6,
             "T": 9
           }
+        },
+        {
+          "description": "puzzle with ten letters and 41 addends",
```

Comment:
I don't understand the "41 addends" part of this description. I see 199 addends. Please clarify what "41 addends" means

Reply:
Sorry, made a new PR
https://github.com/exercism/problem-specifications/pull/1045